File: huntkingdom2/src/evenement/Entity/evenement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package evenement.Entity;

import java.util.Date;
import javafx.scene.image.ImageView;

/**
 *
 * @author hazem
 */
public class evenement {
    private int id_event;
    private String titre_event;
     private String image;
    private Date date_debut_event;
    private Date date_fin_event;
    private int nb_place_event;
    private ImageView photo;

    public ImageView getPhoto() {
        return photo;
    }

    public void setPhoto(ImageView photo) {
        this.photo = photo;
    }

    public evenement(int id_event, String titre_event,String image, Date date_debut_event, Date date_fin_event, int nb_place_event,ImageView photo) {
        this.id_event = id_event;
        this.titre_event = titre_event;
        this.image = image;
        this.date_debut_event = date_debut_event;
        this.date_fin_event = date_fin_event;
        this.nb_place_event = nb_place_event;
        this.photo=photo;
    }
        public evenement(int id_event, String titre_event,String image, Date date_debut_event, Date date_fin_event, int nb_place_event) {
        this.id_event = id_event;
        this.titre_event = titre_event;
        this.image = image;
        this.date_debut_event = date_debut_event;
        this.date_fin_event = date_fin_event;
        this.nb_place_event = nb_place_event;
       
    }

    public evenement(String titre_event,String image, Date date_debut_event, Date date_fin_event, int nb_place_event) {
        this.titre_event = titre_event;
        this.image = image;
        this.date_debut_event = date_debut_event;
        this.date_fin_event = date_fin_event;
        this.nb_place_event = nb_place_event;
    }

    public int getId_event() {
        return id_event;
    }

    public String getTitre_event() {
        return titre_event;
    }

    public Date getDate_debut_event() {
        return date_debut_event;
    }

    public Date getDate_fin_event() {
        return date_fin_event;
    }

    public int getNb_place_event() {
        return nb_place_event;
    }

    public void setId_event(int id_event) {
        this.id_event = id_event;
    }

    public void setTitre_event(String titre_event) {
        this.titre_event = titre_event;
    }

    public void setDate_debut_event(Date date_debut_event) {
        this.date_debut_event = date_debut_event;
    }

    public void setDate_fin_event(Date date_fin_event) {
        this.date_fin_event = date_fin_event;
    }

    public void setNb_place_event(int nb_place_event) {
        this.nb_place_event = nb_place_event;
    }

    public String getImage() {
        return image;
    }

    @Override
    public String toString() {
        return "evenement{" + "id_event=" + id_event + ", titre_event=" + titre_event + ", image=" + image + ", date_debut_event=" + date_debut_event + ", date_fin_event=" + date_fin_event + ", nb_place_event=" + nb_place_event + ", photo=" + photo + '}';
    }

  

    public void setImage(String image) {
        this.image = image;
    }

    
    
    
    
}

File: symfony/project/sympfony/src/HuntkingdomBundle/Entity/Evenement.php
<?php

namespace HuntkingdomBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Evenement
 *
 * @ORM\Table(name="evenement")
 * @ORM\Entity
 */
class Evenement
{
    /**
     * @return string
     */
    public function getTitreEvent()
    {
        return $this->titreEvent;
    }

    /**
     * @param string $titreEvent
     */
    public function setTitreEvent($titreEvent)
    {
        $this->titreEvent = $titreEvent;
    }

    /**
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * @param string $image
     */
    public function setImage($image)
    {
        $this->image = $image;
    }

    /**
     * @return \DateTime
     */
    public function getDateDebutEvent()
    {
        return $this->dateDebutEvent;
    }

    /**
     * @param \DateTime $dateDebutEvent
     */
    public function setDateDebutEvent($dateDebutEvent)
    {
        $this->dateDebutEvent = $dateDebutEvent;
    }

    /**
     * @return \DateTime
     */
    public function getDateFinEvent()
    {
        return $this->dateFinEvent;
    }

    /**
     * @param \DateTime $dateFinEvent
     */
    public function setDateFinEvent($dateFinEvent)
    {
        $this->dateFinEvent = $dateFinEvent;
    }

    /**
     * @return int
     */
    public function getNbPlaceEvent()
    {
        return $this->nbPlaceEvent;
    }

    /**
     * @param int $nbPlaceEvent
     */
    public function setNbPlaceEvent($nbPlaceEvent)
    {
        $this->nbPlaceEvent = $nbPlaceEvent;
    }
    /**
     * @return int
     */
    public function getIdEvent()
    {
        return $this->idEvent;
    }

    /**
     * @param int $idEvent
     */
    public function setIdEvent($idEvent)
    {
        $this->idEvent = $idEvent;
    }
    /**
     * @var integer
     *
     * @ORM\Column(name="id_event", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idEvent;

    /**
     * @var string
     *
     * @ORM\Column(name="titre_event", type="string", length=50, nullable=false)
     */
    private $titreEvent;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=255, nullable=false)
     */
    private $image;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_debut_event", type="date", nullable=false)
     */
    private $dateDebutEvent;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_fin_event", type="date", nullable=false)
     */
    private $dateFinEvent;

    /**
     * @var integer
     *
     * @ORM\Column(name="nb_place_event", type="integer", nullable=false)
     */
    private $nbPlaceEvent;


}

